### PR TITLE
K fidan1/h4 i 14 create chapters collection

### PR DESCRIPTION
**Pull request by kemalfidan**
A lot of important things in this CR. It pretty much sets the workflow for querying our backend. `requests/` is used from pages or components to fetch data. The requests then hit an api endpoint under `api/` which then route to actions under `server/`. Some stuff still need to be removed like commented out code and chapter/byName so I'll be sure to do that in a next PR. 

I also fixed the footer styling in this PR since the reloads weren't loading the styling properly.

### CODE REVIEW

File: server/index.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  //   })
+  //   .catch(error => {
+  //     console.error("Database connection failed.");
+  //     console.error(" > " + error);
+
+  //     throw error;
+  //   });
+
+    await mongoose
+    .connect("mongodb+srv://kfidan:ZiPwdaph0GnqjlbF@cluster0.wzdut.mongodb.net/hack4impact?retryWrites=true&w=majority", {
```
Comment (tbatk) [inline server/index.ts]: Not sure how secure it is to leave this visible in the repo, but I think it contains the password to the mongo cluster in the connection string, so we might need to store this in an environment variable.

**Review by tbatk — APPROVED** [review]
Everything looks good. The only thing that might need to be changed is the connection string in the server index file.

**Review by rluberto — APPROVED** [review]
Looks good! The only problem I had was with the footer not loading because the f is capital in your import in the index.tsx but I checked the develop and the footer is lowercase in the import there so it should work.

File: server/index.ts (hunk 1 above)
Comment (kemalfidan) [reply server/index.ts]: You're right. This one connects to a dummy collection under my personal acc but i'll change it before the merge

**Comment by kemalfidan** [comment]
I thought I changed the import index.tsx to import Footer instead of footer. I'll double check after the merge though

**Pull request merged**